components/common/TableHeader.vue

```diff
@@ -35,27 +41,16 @@ export default {
       default: true,
     },
   },
-  data() {
-    return {
-      activeProperty: '',
-    }
-  },
-  computed: {
-    sortOrder() {
-      return this.sort.order
-    },
-  },
   methods: {
     orderBy(property) {
-      this.activeProperty = property
-      if (this.sort.property === property) {
-        if (this.sort.order === `asc`) {
-          this.sort.order = `desc`
+      if (this.sortProperty === property) {
+        if (this.sortOrder === `asc`) {
+          this.sortOrder = `desc`
         } else {
-          this.sort.order = `asc`
+          this.sortOrder = `asc`
         }
       } else {
-        this.sort.property = property
+        this.sortProperty = property
```

Comment:
it seems it ends up sorting by VotingPower, since this is the last property in the Array

![image](https://user-images.githubusercontent.com/40721795/99146801-80e11680-267b-11eb-846c-b0e68de86f67.png)

Is this wanted?

By the way, the sort is broken now. I click on the selectors and nothing happens. Debugging

Reply:
no i don't think that's correct. `votingPower` is set intentionally as part of the `sort` object which is passed in as a prop.

Reply:
it defaults to sorting by VotingPower. Should we leave it like that? Which one should be the default sorting property?

Reply:
yes. 

> `votingPower` is set intentionally as part of the sort object which is passed in as a prop.